relayer/strategies.go

```diff
@@ -4,12 +4,17 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"io"
 	"strings"
 	"sync"
 	"time"
 
 	"github.com/avast/retry-go/v4"
 	"github.com/cosmos/ibc-go/v3/modules/core/04-channel/types"
+	cosmosProcessor "github.com/cosmos/relayer/v2/relayer/chains/cosmos"
+	"github.com/cosmos/relayer/v2/relayer/processor"
+	"github.com/cosmos/relayer/v2/relayer/provider"
+	cosmosProvider "github.com/cosmos/relayer/v2/relayer/provider/cosmos"
```

Comment:
Not in scope for this PR, but this is why you'll often see repeated parts of the path in package names, e.g. `.../relayer/chains/cosmoschain` and `.../relayer/provider/cosmosprovider`, so that you can import both without ambiguity and necessary renames.

Import names should be all lowercase, i.e. `cosmosprocessor` and `cosmosprovider`.

Reply:
done